cmd/podman/cleanup.go

```diff
@@ -49,7 +49,7 @@ func cleanupCmd(c *cli.Context) error {
 
 	cleanupContainers, lastError := getAllOrLatestContainers(c, runtime, -1, "all")
 
-	ctx := getContext()
+	ctx := getContext(c)
```

Comment:
it is probably too much work at this point, but shouldn't all the existing `getContext() -> getContext(c)` be part of the other patch?  Otherwise while bisecting the previous patch doesn't build

Reply:
I've reordered the commits so that the library change is before the other. This should work around that issue.